ProjectSidewalk/SidewalkWebpage
# 1374 parameterize city specific data

Resolves #1374

This parameterized Project Sidewalk by city. We will now just need to update the `SIDEWALK_CITY_ID` environment variable, and all the maps should be centered on the correct city, all text on the website should relate to that city, etc. 

List of changes:
1. landing page "Your work is making a difference" section has the name of the city parameterized in two places
1. choropleths on the landing page, /results page, user dashboard, /demo page, /admin map page, /admin analytics page, and /admin user dashboard (/admin/user/<username>) are all now centered on the correct city with an appropriate default zoom set (12 for DC, 14 for Newberg b/c it is so much smaller). note that the admin user dashoard 
1. on the audit page, after the tutorial, the "Let's get started" popup is now parameterized
![lets-get-started-parameterized](https://user-images.githubusercontent.com/6518824/52014627-e1966380-2494-11e9-94c9-9e62555c4f4b.png)
1. on the top-right of the audit page, the city name is now parameterized
![current-neighborhood-paramterized](https://user-images.githubusercontent.com/6518824/52014472-78165500-2494-11e9-9182-91b215fa0493.png)
1. the mission complete map is now focused on the correct city
1. a text reference to DC on the /demo page was parameterized
1. disclaimer on the API page was changed from "We are focusing on collecting data only from Washington, D.C. at the moment." to "We are focusing on collecting data only from Washington, DC and Newberg, OR at the moment."
1. text references to DC on the admin page were changed to "city"
1. a few comments were changed to refer to a generic "city" instead of DC
1. the `street_edge_id` for the tutorial street will be different in each city, even though the street is the same. check to make sure that the tutorial works without any errors. right now the tutorial street is placed in the neighborhood with `region_id=1` which is a quick fix but I don't see a good way around it right now.

The only things I did not update are
1. the press on the landing page was left as-is. hopefully we get some good PR (even in the form of tweets) for Newberg, and we can fill that in later
1. the small maps on the /api page have not been parameterized. once we have more data from Newberg, I want to pick good locations for those maps and parameterize at that time

Testing instructions
1. run `export SIDEWALK_CITY_ID="washington-dc"` in the terminal (I would actually recommend just adding it to your .bashrc file
1. run the code on the 1374 branch and make sure that all of the things listed above seem to be working correclty (the text says "DC" or "Washington" or whatever, and the maps are all centered on DC.
1. I'll send you a database dump for Newberg, OR. You should setup that database in the same way you set up the DC database -- you can just name it "newberg-sidewalk" instead of "sidewalk" so you don't have to delete your "sidewalk" database, then modify your `conf/application.conf` file's `db.default.url` line to say "newberg-sidewalk" instead of "sidewalk"
1. update your .bashrc to have the line `export SIDEWALK_CITY_ID="newberg-or"`
1. test everything again to make sure it has all been updated to say "Newberg" and the maps are centered on the correct city
